src/proxy.rs

```diff
@@ -122,6 +124,10 @@ impl<'a> Proxy<'a> {
         self.request(handle_token, method_name, body).await
     }
 
+    pub async fn version(&self) -> u32 {
```

Comment:
```suggestion
    pub fn version(&self) -> u32 {
```
no longer async & the various awaits are not needed

Reply:
Right